schemachange/cli.py

```diff
@@ -184,8 +195,14 @@ def get_all_scripts_recursively(root_directory, verbose):
       script['script_name'] = file_name
       script['script_full_path'] = file_full_path
       script['script_type'] = script_type
-      script['script_version'] = None if script_type == 'R' else script_name_parts.group(2)
-      script['script_description'] = (repeatable_script_name_parts.group(2) if script_type == 'R' else script_name_parts.group(3)).replace('_', ' ').capitalize()
+      script['script_version'] = None if script_type in ['R', 'D'] else script_name_parts.group(2)
+      if script_type == 'R':
+        script['script_description'] = repeatable_script_name_parts.group(2).replace('_', ' ').capitalize()
+      elif script_type == 'D':
+        script['script_description'] = delta_script_name_parts.group(2).replace('_', ' ').capitalize()
+      else:
+        script_name_parts.group(3).replace('_', ' ').capitalize()
```

Comment:
should be:
`        script['script_description'] = script_name_parts.group(3).replace('_', ' ').capitalize()`

Reply:
updated with new commit